mypy/checker.py

```diff
@@ -2388,23 +2388,32 @@ def find_isinstance_check(node: Node,
 
     Guaranteed to not return None, None. (But may return {}, {})
     """
-    def split_types(current_type: Optional[Type], type_if_true: Optional[Type], expr: Node) \
-            -> Tuple[Optional[Dict[Node, Type]], Optional[Dict[Node, Type]]]:
-        if type_if_true:
+    def conditional_type_map(expr: Node,
```

Comment:
Yeah, I learned last week that I really shouldn't use nested functions like this.  Moved it out.